subprojects/persistence_model/src/main/kotlin/club/staircrusher/infra/persistence/sqldelight/Transaction.kt

```diff
@@ -0,0 +1,20 @@
+package club.staircrusher.infra.persistence.sqldelight
+
+import app.cash.sqldelight.TransactionWithReturn
+import club.staircrusher.stdlib.persistence.Transaction
+
+class Transaction<T>(
```

Comment:
```suggestion
class SqlDelightTransaction<T>(
```
그냥 Transaction하면 넘 헷갈릴 듯 ㅜ

Reply:
근데 얘는 구현체니까 거의 볼일이 없지 않아요?